syntax = "proto3";

option java_package = "ex.grpc";

package chat;

// Defines the service
service Chat {
    // Function invoked to send the request
    rpc sendRequest (ChatRequest) returns (ChatReply) {}
}

// The request message containing requested numbers
message ChatRequest {
    string message = 1;
}

// The response message containing response
message ChatReply {
    string reply = 1;
}
